nautobot/core/views/utils.py

```diff
@@ -286,7 +286,7 @@ def view_changes_not_saved(request, view, current_saved_view):
         return True
     query_dict = request.GET.dict()
 
-    if query_dict.get("table_changes_pending", None):
+    if query_dict.get("table_changes_pending", None) or query_dict.get("all_filters_removed", None):
```

Comment:
I wonder if this should be:

```suggestion
    if "table_changes_pending" in query_dict or "all_filters_removed" in query_dict:
```